sample_apps/run_example.sh

```diff
@@ -508,9 +609,12 @@ function author_policy() {
 
     print_policy
 
-    construct_platform_key_attestation_stmt_sign_it
+    # RESOLVE: X-check v/s instructions. These steps seem to be n/a for Gramine App
```

Comment:
@yelvmw -- We need to talk this thru. I just implemented something per the instructions, but not sure if those are all correct even.